Core/src/IfcGeometryConverter/CurveConverter.h

```diff
@@ -2223,11 +2171,27 @@ namespace OpenInfraPlatform {
 					double yRadius = ellipse->SemiAxis2 * UnitConvert()->getLengthInMeterFactor();
 
 					carve::geom::vector<3> ellipse_center =
-						conicPositionMatrix * carve::geom::VECTOR(0, 0, 0);
+						conicPositionMatrix * carve::geom::VECTOR(0., 0., 0.);
 
 					return ellipse_center + carve::geom::VECTOR(xRadius * cos(angle), yRadius * sin(angle), 0.);

```

Comment:
I hope, I have implemented it right in 004a820. I have calculated this as (left-)multiplication (At least, I think so). Right-multiplication was not possible, since the `return` should be then from type `carve::math::Matrix`

Reply:
Resolved: 7826079